src/vmware/HISTORY.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 5.0.0 (2022-09)
+
+- Update to AVS 2022-05-01 API
+- Add `az vmware cluster list-zones`
+- Add `az vmware workload-network show`
+- Add `az vmware workload-network list`
```

Comment:
Please test `workload-network show` and `list` since I do not think they are actually implemented in the service. They are in the spec so they are seen as resources for the SDKs and ARM Bicep. Probably best to remove them.

Reply:
Good catch. I tested in prod and it says that those requests are not implemented. Removed them and the tests I wrote.